spec/rails_helper.rb

```diff
@@ -2,6 +2,7 @@
 SimpleCov.start 'rails' do
   add_group 'Queries', 'app/queries'
   add_group 'Validators', 'app/validators'
+  minimum_coverage 98
```

Comment:
Should we add `add_group 'Presenters', 'app/presenters'` for completeness?

Reply:
Yes! good point